action.yml

```
@@ -22,7 +22,7 @@ runs:
       case ${{ inputs.version }} in
       tip)
         echo "Installing ko using go install"
-        go install github.com/google/ko@main
+        go install github.com/ko-build/ko@main
```

```suggestion
        go install github.com/google/ko@main
```

This one is still the old URL until we change the importpath.